src/Battle.php

```diff
@@ -0,0 +1,211 @@
+<?php
+declare(strict_types=1);
+
+namespace LotGD\Core;
+
+use LotGD\Core\{
+    DiceBag,
+    Exceptions\ArgumentException,
+    Exceptions\BattleIsOverException,
+    Exceptions\BattleNotOverException,
+    Models\FighterInterface
+};
+
+/**
+ * Description of Battle
+ */
+class Battle
+{
+    const DAMAGEROUND_PLAYER = 0b01;
+    const DAMAGEROUND_MONSTER = 0b10;
+    const DAMAGEROUND_BOTH = 0b11;
+    
```

Comment:
This is fine for now but keep in mind there are mods that add battles between more than two combatants.